README.md

```diff
@@ -898,6 +898,21 @@ macro:
     mean: 25.911653497483996
 ```
 
+### SPARQL queries comparison
+
+The algoritm iterates over all of the subsets of the actual result in which the number of column is equal to the number of columns in the reference result.
+For each subset it tries to find a permutation in columns in which the two results match. In the case two results have different number of result the output is considered false. Optional columns are skipped. Also support for floating-point numbers exist but for special types like duration is not yet added.
```

Comment:
`permutation of`? ALso, I totally don't get this sentence `In the case two results have different number of result the output is considered false`